pkgs/applications/audio/squeezelite/default.nix

```diff
@@ -44,7 +53,10 @@ stdenv.mkDerivation {
   };
 
   buildInputs = [ flac libmad libvorbis mpg123 ]
-    ++ lib.singleton (if pulseSupport then libpulseaudio else alsa-lib)
+    ++ optional pulseSupport libpulseaudio
+    ++ optional alsaSupport alsa-lib
+    ++ optional portaudioSupport portaudio
+    ++ optionals stdenv.isDarwin [CoreVideo VideoDecodeAcceleration CoreAudio AudioToolbox AudioUnit Carbon]
```

Comment:
```suggestion
    ++ optionals stdenv.isDarwin [ CoreAudio VideoDecodeAcceleration AudioToolbox AudioUnit Carbon ]
```

Reply:
Did you mean to change the frameworks here, or just add space padding inside the list?

Reply:
Framework list lifted from upstream here: https://github.com/ralph-irving/squeezelite/blob/575b59398dd19945d2038474062240fc527407fb/Makefile.m1#L6

Reply:
Nope, just to remove the duplicated `CoreAudio` here.

Reply:
Oh, my bad.